content/learning-how-to-teach-coders/quizzes/_index.md

```diff
@@ -1,5 +1,45 @@
 ---
-_db_id: 649
```

Comment:
why did you delete this line? 

If _db_id is there: dont do anything to it

Reply:
@kaleem99 thats my bad, i didnt realise the _db_id was autogenerated as I thought this was a completely new page, apologies